src/css/components/_c-checklist.scss

```diff
@@ -282,3 +282,11 @@ margin-left: 1rem;
 		padding-left: map-get($_checklist-inset, large);
 	}
 }
+
+.u-title-case {
+	text-transform: lowercase;
+}
+
+.u-title-case::first-letter {
+	text-transform: uppercase;
+}
```

Comment:
We could also refactor this to be a little more self-contained:

```scss
.u-title-case {
	text-transform: lowercase;

	&::first-letter {
		text-transform: uppercase;
	}
}
```

Also, this is a clever little utility that I'm definitely stealing for my own personal work.